actionpack/test/controller/filters_test.rb

```diff
@@ -15,6 +15,7 @@ def before_actions
   end
 end
 
+# <tt>filters</tt> refers to action callbacks, previously being call 'filters'.
```

Comment:
Is this helpful? Correct comment syntax?  My intent is to call attention to the entire test file and a number of  variables still refer to action callbacks as `filters`.  I assume that any renaming of the file or variables would be done in a separate PR.